### DISCUSSION TITLE
Gulp processing very slow?

### DISCUSSION THREAD

**Discussion by lilpxl** [understrap/understrap-child#383] [General]
Hi everyone!

I used Understrap religiously for a number of years - its been great and want to start this off saying how appreciative I am to everyone involved in  this project.

I recently started building my first site with the new version of Understrap (after it was purchased by the current ownsers) and am noticing that the new `npm run watch` process is significantly slower than the old `gulp watch-bs` process. At first I figured it would not be a big deal but after spending some hours working I am realizing it makes my workflow significantly less efficient.

The old `gulp watch-bs` process was near instantaneous, so I could make little changes and see them appear - but now I am having to wait a solid ten seconds simply to see a small CSS tweak.

Is there something I am missing? Is there a way to revert or prune out the processes to cut that time down?

Thank you everyone!

**Comment by bacoords** [comment]
I understand- the difference from node-sass to sass is definitely noticeable. You can read more about the discussion here: https://github.com/understrap/understrap/issues/1428

We've also offered this as a solution, although we can can't merge this in because it doesn't work for all users. That said you may be able to use it yourself: https://github.com/understrap/understrap/issues/1598
